bin/test-sbt-1.0

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+. "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/scriptLib"
+
+SCALA_VERSION="2.12.4"
+
+runSbtNoisy "++${SCALA_VERSION} publishLocal"
+runSbtNoisy "++${SCALA_VERSION} scripted"
```

Comment:
I'm not sure why `++` isn't properly changing the scala version of `sbt-scripted-tools`. I even tried using sbt-doge's `wow 2.124` but it also doesn't work.

To unblock this I would suggest just adding

```scala
'set scalaVersion in `sbt-scripted-tools` := "2.12.4"'
```

after `++` in the two executions above.

Reply:
Hey @dwijnand, no game. But it goes further and fails when running the scripted tests instead. Relevant section of the output:

```
$ sbt ++2.12.4 'set scalaVersion in `sbt-scripted-tools` := "2.12.4"' scripted
[info] Loading global plugins from /Users/marcospereira/.sbt/0.13/plugins
[info] Loading project definition from /Users/marcospereira/Lightbend/lagom/lagom/project/project
[info] Loading project definition from /Users/marcospereira/Lightbend/lagom/lagom/project
[info] Resolving key references (56750 settings) ...

...

[info] Set current project to lagom (in build file:/Users/marcospereira/Lightbend/lagom/lagom/)
[info] Setting version to 2.12.4
[info] Reapplying settings...

...

[info] Set current project to lagom (in build file:/Users/marcospereira/Lightbend/lagom/lagom/)
[info] Defining sbt-scripted-tools/*:scalaVersion
[info] The new value will be used by sbt-scripted-tools/*:allDependencies, sbt-scripted-tools/*:evicted and 29 others.
[info] 	Run `last` for details.
[info] Reapplying settings...

...

[info] Set current project to lagom (in build file:/Users/marcospereira/Lightbend/lagom/lagom/)
[info] Packaging /Users/marcospereira/Lightbend/lagom/lagom/dev/sbt-scripted-tools/target/scala-2.12/sbt-1.0/lagom-sbt-scripted-tools-1.4.0-SNAPSHOT-sources.jar ...
[warn] Credentials file /Users/marcospereira/.bintray/.credentials does not exist
[info] Done packaging.
[warn] Credentials file /Users/marcospereira/.bintray/.credentials does not exist
[info] Updating {file:/Users/marcospereira/Lightbend/lagom/lagom/}build-tool-support...
[info] Wrote /Users/marcospereira/Lightbend/lagom/lagom/dev/sbt-plugin/target/scala-2.12/sbt-1.0/lagom-sbt-plugin-1.4.0-SNAPSHOT.pom
[info] Done updating.
[warn] Credentials file /Users/marcospereira/.bintray/.credentials does not exist
[info] Wrote /Users/marcospereira/Lightbend/lagom/lagom/dev/sbt-scripted-tools/target/scala-2.12/sbt-1.0/lagom-sbt-scripted-tools-1.4.0-SNAPSHOT.pom
[warn] Found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[warn]
[warn] 	* com.google.guava:guava:23.0 is selected over 19.0
[warn] 	    +- io.methvin:directory-watcher:0.3.0                 (depends on 23.0)
[warn] 	    +- com.datastax.cassandra:cassandra-driver-core:3.2.0 (depends on 19.0)
[warn]
[warn] Run 'evicted' to see detailed eviction warnings
[warn] Credentials file /Users/marcospereira/.bintray/.credentials does not exist
[info] Wrote /Users/marcospereira/Lightbend/lagom/lagom/dev/sbt-scripted-library/target/scala-2.11/lagom-sbt-scripted-library_2.11-1.4.0-SNAPSHOT.pom
[info] Wrote /Users/marcospereira/Lightbend/lagom/lagom/dev/service-registry/service-locator/target/scala-2.12/lagom-service-locator_2.12-1.4.0-SNAPSHOT.pom
[info] Updating {file:/Users/marcospereira/Lightbend/lagom/lagom/}sbt-plugin...
[info] Done updating.
[info] :: delivering :: com.lightbend.lagom#lagom-service-locator_2.12;1.4.0-SNAPSHOT :: 1.4.0-SNAPSHOT :: integration :: Tue Nov 21 19:12:54 BRST 2017
[info] 	delivering ivy file to /Users/marcospereira/Lightbend/lagom/lagom/dev/service-registry/service-locator/target/scala-2.12/ivy-1.4.0-SNAPSHOT.xml

...

[info] :: delivering :: com.lightbend.lagom#lagom-sbt-scripted-library_2.11;1.4.0-SNAPSHOT :: 1.4.0-SNAPSHOT :: integration :: Tue Nov 21 19:12:54 BRST 2017
[info] 	delivering ivy file to /Users/marcospereira/Lightbend/lagom/lagom/dev/sbt-scripted-library/target/scala-2.11/ivy-1.4.0-SNAPSHOT.xml
[info] 	published lagom-service-locator_2.12 to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-service-locator_2.12/1.4.0-SNAPSHOT/poms/lagom-service-locator_2.12.pom
[info] 	published lagom-service-locator_2.12 to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-service-locator_2.12/1.4.0-SNAPSHOT/jars/lagom-service-locator_2.12.jar
[info] 	published lagom-service-locator_2.12 to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-service-locator_2.12/1.4.0-SNAPSHOT/srcs/lagom-service-locator_2.12-sources.jar
[info] 	published lagom-service-locator_2.12 to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-service-locator_2.12/1.4.0-SNAPSHOT/docs/lagom-service-locator_2.12-javadoc.jar
[info] 	published ivy to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-service-locator_2.12/1.4.0-SNAPSHOT/ivys/ivy.xml
[info] :: delivering :: com.lightbend.lagom#lagom-sbt-plugin;1.4.0-SNAPSHOT :: 1.4.0-SNAPSHOT :: integration :: Tue Nov 21 19:12:54 BRST 2017
[info] 	delivering ivy file to /Users/marcospereira/Lightbend/lagom/lagom/dev/sbt-plugin/target/scala-2.12/sbt-1.0/ivy-1.4.0-SNAPSHOT.xml
[info] 	published lagom-sbt-scripted-library_2.11 to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-sbt-scripted-library_2.11/1.4.0-SNAPSHOT/poms/lagom-sbt-scripted-library_2.11.pom
[info] Formatting 1 Scala source {file:/Users/marcospereira/Lightbend/lagom/lagom/}build-tool-support(compile) ...
[info] 	published lagom-sbt-scripted-library_2.11 to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-sbt-scripted-library_2.11/1.4.0-SNAPSHOT/jars/lagom-sbt-scripted-library_2.11.jar
[info] 	published lagom-sbt-scripted-library_2.11 to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-sbt-scripted-library_2.11/1.4.0-SNAPSHOT/srcs/lagom-sbt-scripted-library_2.11-sources.jar
[info] 	published lagom-sbt-scripted-library_2.11 to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-sbt-scripted-library_2.11/1.4.0-SNAPSHOT/docs/lagom-sbt-scripted-library_2.11-javadoc.jar
[info] 	published ivy to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-sbt-scripted-library_2.11/1.4.0-SNAPSHOT/ivys/ivy.xml
[info] Updating {file:/Users/marcospereira/Lightbend/lagom/lagom/}sbt-scripted-tools...
[info] Reformatted 1 Scala source {file:/Users/marcospereira/Lightbend/lagom/lagom/}build-tool-support(compile).
[info] Packaging /Users/marcospereira/Lightbend/lagom/lagom/dev/sbt-plugin/target/scala-2.12/sbt-1.0/lagom-sbt-plugin-1.4.0-SNAPSHOT.jar ...
[info] Done packaging.
[info] Done updating.

...

[info] 	published lagom-sbt-plugin to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-sbt-plugin/scala_2.12/sbt_1.0/1.4.0-SNAPSHOT/jars/lagom-sbt-plugin.jar
[info] 	published lagom-sbt-plugin to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-sbt-plugin/scala_2.12/sbt_1.0/1.4.0-SNAPSHOT/srcs/lagom-sbt-plugin-sources.jar
[info] 	published lagom-sbt-plugin to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-sbt-plugin/scala_2.12/sbt_1.0/1.4.0-SNAPSHOT/docs/lagom-sbt-plugin-javadoc.jar
[info] 	published ivy to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-sbt-plugin/scala_2.12/sbt_1.0/1.4.0-SNAPSHOT/ivys/ivy.xml
[info] :: delivering :: com.lightbend.lagom#lagom-sbt-scripted-tools;1.4.0-SNAPSHOT :: 1.4.0-SNAPSHOT :: integration :: Tue Nov 21 19:12:58 BRST 2017
[info] 	delivering ivy file to /Users/marcospereira/Lightbend/lagom/lagom/dev/sbt-scripted-tools/target/scala-2.12/sbt-1.0/ivy-1.4.0-SNAPSHOT.xml
[info] Main Scala API documentation to /Users/marcospereira/Lightbend/lagom/lagom/dev/sbt-scripted-tools/target/scala-2.12/sbt-1.0/api...
[info] Packaging /Users/marcospereira/Lightbend/lagom/lagom/dev/sbt-scripted-tools/target/scala-2.12/sbt-1.0/lagom-sbt-scripted-tools-1.4.0-SNAPSHOT.jar ...
[info] Done packaging.
model contains 11 documentable templates
[info] Main Scala API documentation successful.
[info] Packaging /Users/marcospereira/Lightbend/lagom/lagom/dev/sbt-scripted-tools/target/scala-2.12/sbt-1.0/lagom-sbt-scripted-tools-1.4.0-SNAPSHOT-javadoc.jar ...
[info] Done packaging.
[info] 	published lagom-sbt-scripted-tools to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-sbt-scripted-tools/scala_2.12/sbt_1.0/1.4.0-SNAPSHOT/poms/lagom-sbt-scripted-tools.pom
[info] 	published lagom-sbt-scripted-tools to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-sbt-scripted-tools/scala_2.12/sbt_1.0/1.4.0-SNAPSHOT/jars/lagom-sbt-scripted-tools.jar
[info] 	published lagom-sbt-scripted-tools to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-sbt-scripted-tools/scala_2.12/sbt_1.0/1.4.0-SNAPSHOT/srcs/lagom-sbt-scripted-tools-sources.jar
[info] 	published lagom-sbt-scripted-tools to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-sbt-scripted-tools/scala_2.12/sbt_1.0/1.4.0-SNAPSHOT/docs/lagom-sbt-scripted-tools-javadoc.jar
[info] 	published ivy to /Users/marcospereira/.ivy2/local/com.lightbend.lagom/lagom-sbt-scripted-tools/scala_2.12/sbt_1.0/1.4.0-SNAPSHOT/ivys/ivy.xml
Running sbt-plugin / run
[info] Getting org.scala-sbt sbt 1.0.2 ...
[info] :: retrieving :: org.scala-sbt#boot-app
[info] 	confs: [default]
[info] 	69 artifacts copied, 0 already retrieved (22000kB/189ms)
[info] Getting Scala 2.12.3 (for sbt)...
[info] :: retrieving :: org.scala-sbt#boot-scala
[info] 	confs: [default]
[info] 	5 artifacts copied, 0 already retrieved (19004kB/50ms)
[info] [info] Updated file /private/var/folders/rj/gc0lq1394_q3f7nz5kf159yr0000gn/T/sbt_1ed61cc0/run/project/build.properties: set sbt.version to 1.0.2
[info] [info] Loading settings from plugins.sbt ...
[info] [info] Loading project definition from /private/var/folders/rj/gc0lq1394_q3f7nz5kf159yr0000gn/T/sbt_1ed61cc0/run/project
[info] [info] Updating {file:/private/var/folders/rj/gc0lq1394_q3f7nz5kf159yr0000gn/T/sbt_1ed61cc0/run/project/}run-build...
[info] [warn] Choosing local for com.lightbend.lagom#lagom-sbt-scripted-tools;1.4.0-SNAPSHOT
[info] [warn] Choosing local for com.lightbend.lagom#lagom-sbt-plugin;1.4.0-SNAPSHOT
[info] [warn] Choosing local for com.lightbend.lagom#lagom-build-tool-support;1.4.0-SNAPSHOT
[info] [info] Done updating.

...

[info] [error] Modules were resolved with conflicting cross-version suffixes in {file:/private/var/folders/rj/gc0lq1394_q3f7nz5kf159yr0000gn/T/sbt_1ed61cc0/run/project/}run-build:
[info] [error]    com.lightbend.play:play-file-watch _2.10, _2.12
[info] [error]    com.github.pathikrit:better-files _2.10, _2.12
[info] [error] java.lang.RuntimeException: Conflicting cross-version suffixes in: com.lightbend.play:play-file-watch, com.github.pathikrit:better-files
[info] [error] 	at scala.sys.package$.error(package.scala:27)
[info] [error] 	at sbt.librarymanagement.ConflictWarning$.processCrossVersioned(ConflictWarning.scala:39)
[info] [error] 	at sbt.librarymanagement.ConflictWarning$.apply(ConflictWarning.scala:19)
[info] [error] 	at sbt.Classpaths$.$anonfun$ivyBaseSettings$64(Defaults.scala:1971)
[info] [error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:44)
[info] [error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:42)
[info] [error] 	at sbt.std.Transform$$anon$4.work(System.scala:64)
[info] [error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:257)
[info] [error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[info] [error] 	at sbt.Execute.work(Execute.scala:266)
[info] [error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:257)
[info] [error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:167)
[info] [error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:32)
[info] [error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[info] [error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[info] [error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[info] [error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[info] [error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[info] [error] 	at java.lang.Thread.run(Thread.java:748)
[info] [error] (*:update) Conflicting cross-version suffixes in: com.lightbend.play:play-file-watch, com.github.pathikrit:better-files
[info] Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore?
[error] x sbt-plugin / run
[error]  Cause of test exception: {line 19}  Command failed: Remote sbt initialization failed
```

Reply:
By the way, I was trying to use `wow` too, but no luck.